doc/manual/index.rst

```diff
@@ -1,7 +1,7 @@
 .. _manual:
 
-Manual
-======
+User Manual
```

Comment:
I think it's been called `Manual` for a while, so I don't really want to remove that name from it entirely.